detections/endpoint/remcos_rat_file_creation_in_remcos_folder.yml

```diff
@@ -12,8 +12,8 @@ description: This search is to detect file creation in remcos folder in appdata
   clipboard grabbing and audio recording.
 search: '|tstats `security_content_summariesonly` count min(_time) as firstTime max(_time)
   as lastTime FROM datamodel=Endpoint.Filesystem where Filesystem.file_name IN ("*.dat")
-  Filesystem.file_path = "*\\remcos\\*" by _time Filesystem.file_name Filesystem.file_path
-  Filesystem.dest Filesystem.file_create_time | `drop_dm_object_name(Processes)` |
+  Filesystem.file_path = "*\\remcos\\*" by _time Filesystem.file_name c.file_path
```

Comment:
Looks like something went wrong here when the MR was copied, this should still be Filesystem, not c

Reply:
thank you for reviewing! it was my mistake, that spl is fixed now!